### ISSUE TITLE
Consolidate technology & security pages

### ISSUE THREAD

**Issue by karareinsel** [cloud-gov/cg-site#1625]
- [ ] [What the cloud.gov PaaS offers](https://cloud.gov/docs/technology/responsibilities/) - Lots of overlap with What is cloud.gov page. Need to confirm if deprecation policy is still accurate. The deprecation policy might be better located with our Customer Service Objectives. Also, the chart illustrating the 3 scenarios feels bulky and the examples are hard to follow. 
- [ ] [How cloud.gov helps teams comply with requirements ](https://cloud.gov/docs/security/conforming-federal-security-regulations/)- Content overlaps with Overview/FedRAMP authorized, Overview/The benefits of cloud.gov, and Overview/What is cloud.gov. 
- [ ] [Infrastructure under cloud.gov](https://cloud.gov/docs/technology/iaas/) - Remove "In principle ... extending the marketplace." and the following sections: Other IaaS vendors we can support and If you're an IaaS vendor

**Comment by markdboyd** [comment]
Repo is being archived, closing this issue.
